packages/neuron-wallet/src/controllers/dao.ts

```diff
@@ -1,13 +1,15 @@
-import { Cell, OutPoint, TransactionWithoutHash } from 'types/cell-types'
 import { ServiceHasNoResponse, IsRequired } from 'exceptions'
 import { ResponseCode } from 'utils/const'
 import AddressesService from 'services/addresses'
 import CellsService from 'services/cells'
 import LockUtils from 'models/lock-utils'
 import TransactionSender from 'services/transaction-sender'
+import OutPoint from 'models/chain/out-point'
+import Output from 'models/chain/output'
+import Transaction from 'models/chain/transaction'
 
 export default class DaoController {
-  public async getDaoCells(params: Controller.Params.GetDaoCellsParams): Promise<Controller.Response<Cell[]>> {
+  public async getDaoCells(params: Controller.Params.GetDaoCellsParams): Promise<Controller.Response<Output[]>> {
```

Comment:
Get cells or outputs?

Reply:
Cell is alias to outputs, and CKB also using them mixed.

Reply:
Huh, CKB using them mixed doesn't mean we have to follow that misleading style 😹

Reply:
Rename to `Cell` in this file.